platform/admin/ui/src/main/webapp/js/views/installer/AnonClaims.view.js

```diff
@@ -0,0 +1,414 @@
+            this.configuration = options.configuration;
+            this.model.set('curProfile', this.configuration.get('properties').get('profile'));
+        },
+        updateValues: function (e) {
+            var profileName = e.currentTarget[e.currentTarget.selectedIndex].label;
+            var profile = this.model.get('availableProfiles')[profileName];
+            this.configuration.get('properties').set('attributes', profile);
+            this.configuration.get('properties').set('profile', profileName);
+            this.model.trigger('profileChanged');
+        },
+        onRender: function () {
+            this.$('select').multiselect('refresh');
+        }
+    });
+
+    var AnonClaimsMultiValuedEntry = Marionette.ItemView.extend({
+        template: 'anonClaimsList',
+        tagName: 'div',
+        className: 'row-container-div',
+        initialize: function () {
+            this.modelBinder = new Backbone.ModelBinder();
+            this.model.validate = this.validate;
+        },
+        events: {
+            "click .minus-button": "minusButton",
+            "click .editable-list-button": "showList",
+            "click .editable-list-item": "selectItem",
+            "mouseover .editable-list-item": "highlightItem"
+        },
+        modelEvents: {
+            "change": "render"
+        },
+        minusButton: function () {
+            this.model.collection.remove(this.model);
+        },
+        onRender: function () {
+            var bindings = Backbone.ModelBinder.createDefaultBindings(this.el, 'name');
+            this.modelBinder.bind(this.model, this.$el, bindings);
+            this.model.set('showWarning', _.contains(this.model.get('immutableClaims'), this.model.get('claimName')));
+
+        },
+        onClose: function () {
+            this.remove();
+            this.unbind();
+        },
+        showList: function (e) {
+            e.stopPropagation();
+            var list = this.$('.editable-list');
+            var input = this.$('.editable-list-input');
+            list.css({'top': input.outerHeight(), 'left': '0px', 'min-width': input.outerWidth()});
+            list.toggle();
+        },
+        selectItem: function (e) {
+
+            var newValue = this.$(e.target).text();
+            this.$('.editable-list-input').val(newValue);
+            this.$('.editable-list-item').removeClass('selected-list-item');
+            this.$(e.target).addClass('selected-list-item');
+            this.model.set('claimName', newValue);
+            this.$('.editable-list').hide();
+        },
+        highlightItem: function (e) {
+            this.$('.editable-list-item').removeClass('highlighted-list-item');
+            this.$(e.target).addClass('highlighted-list-item');
+        }
+    });
+
+    var AnonClaimsMultiValueCollection = Marionette.CollectionView.extend({
+        itemView: AnonClaimsMultiValuedEntry,
+        tagName: 'div',
+        modelEvents: {
+            "change": "render"
```

Comment:
You shouldn't need the model event here. A collection view will automatically add and remove items from its collection.